Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml

```diff
@@ -26,5 +26,8 @@
                 <Button Text="Write Log" InputTransparent="true" Clicked="WriteLog"/>
             </ViewCell>
         </TableSection>
+        <TableSection Title="Logging">
+            <EntryCell Label="User Id" x:Name="UserIdEntryCell" HorizontalTextAlignment="=End" Completed="UserIdCompleted"/>
```

Comment:
I get a crash on your branch with this change that I don't have with develop  branch:

```
[mono] System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ArgumentException: Requested value '=End' was not found.
[mono]   at System.Enum+EnumResult.SetFailure (System.Enum+ParseFailureKind failure, System.String failureMessageID, System.Object failureMessageFormatArgument) [0x00023] in <d4a23bbd2f544c30a48c44dd622ce09f>:0 
[mono]   at System.Enum.TryParseEnum (System.Type enumType, System.String value, System.Boolean ignoreCase, System.Enum+EnumResult& parseResult) [0x0017a] in <d4a23bbd2f544c30a48c44dd622ce09f>:0 
[mono]   at System.Enum.Parse (System.Type enumType, System.String value, System.Boolean ignoreCase) [0x00010] in <d4a23bbd2f544c30a48c44dd622ce09f>:0 
[mono]   at System.Enum.Parse (System.Type enumType, System.String value) [0x00000] in <d4a23bbd2f544c30a48c44dd622ce09f>:0 
[mono]   at Xamarin.Forms.Xaml.TypeConversionExtensions.ConvertTo (System.Object value, System.Type toType, System.Func`1[TResult] getConverter, System.IServiceProvider serviceProvider) [0x000d2] in D:\agent_work\1\s\Xamarin.Forms.Core\Xaml\TypeConversionExtensions.cs:134 
[mono]   at Xamarin.Forms.Xaml.TypeConversionExtensions.ConvertTo (System.Object value, System.Type toType, System.Func`1[TResult] minfoRetriever, System.IServiceProvider serviceProvider) [0x00021] in D:\agent_work\1\s\Xamarin.Forms.Core\Xaml\TypeConversionExtensions.cs:76 
[mono]   at Xamarin.Forms.Xaml.ApplyPropertiesVisitor.TrySetValue (System.Object element, Xamarin.Forms.BindableProperty property, System.Boolean attached, System.Object value, System.Xml.IXmlLineInfo lineInfo, Xamarin.Forms.Xaml.Internals.XamlServiceProvider serviceProvider, System.Exception& exception) [0x00075] in D:\agent_work\1\s\Xamarin.Forms.Xaml\ApplyPropertiesVisitor.cs:442 
[mono]   at Xamarin.Forms.Xaml.ApplyPropertiesVisitor.SetPropertyValue (System.Object xamlelement, Xamarin.Forms.Xaml.XmlName propertyName, System.Object value, System.Object rootElement, Xamarin.Forms.Xaml.INode node, Xamarin.Forms.Xaml.HydrationContext context, System.Xml.IXmlLineInfo lineInfo) [0x00077] in D:\agent_work\1\s\Xamarin.Forms.Xaml\ApplyPropertiesVisitor.cs:330 
[mono]   at Xamarin.Forms.Xaml.ApplyPropertiesVisitor.Visit (Xamarin.Forms.Xaml.ValueNode node, Xamarin.Forms.Xaml.INode parentNode) [0x0005d] in D:\agent_work\1\s\Xamarin.Forms.Xaml\ApplyPropertiesVisitor.cs:52 
[mono]   at Xamarin.Forms.Xaml.ValueNode.Accept (Xamarin.Forms.Xaml.IXamlNodeVisitor visitor, Xamarin.Forms.Xaml.INode parentNode) [0x00000] in D:\agent_work\1\s\Xamarin.Forms.Xaml\XamlNode.cs:86 
[mono]   at Xamarin.Forms.Xaml.ElementNode.Accept (Xamarin.Forms.Xaml.IXamlNodeVisitor visitor, Xamarin.Forms.Xaml.INode parentNode) [0x00043] in D:\agent_work\1\s\Xamarin.Forms.Xaml\XamlNode.cs:143 
[mono]   at Xamarin.Forms.Xaml.ElementNode.Accept (Xamarin.Forms.Xaml.IXamlNodeVisitor visitor, Xamarin.Forms.Xaml.INode parentNode) [0x00078] in D:\agent_work\1\s\Xamarin.Forms.Xaml\XamlNode.cs:145 
[mono]   at Xamarin.Forms.Xaml.ElementNode.Accept (Xamarin.Forms.Xaml.IXamlNodeVisitor visitor, Xamarin.Forms.Xaml.INode parentNode) [0x00078] in D:\agent_work\1\s\Xamarin.Forms.Xaml\XamlNode.cs:145 
[mono]   at Xamarin.Forms.Xaml.RootNode.Accept (Xamarin.Forms.Xaml.IXamlNodeVisitor visitor, Xamarin.Forms.Xaml.INode parentNode) [0x00078] in D:\agent_work\1\s\Xamarin.Forms.Xaml\XamlNode.cs:203 
[mono]   at Xamarin.Forms.Xaml.XamlLoader.Visit (Xamarin.Forms.Xaml.RootNode rootnode, Xamarin.Forms.Xaml.HydrationContext visitorContext) [0x0007b] in D:\agent_work\1\s\Xamarin.Forms.Xaml\XamlLoader.cs:141 
[mono]   at Xamarin.Forms.Xaml.XamlLoader.Load (System.Object view, System.String xaml) [0x0004b] in D:\agent_work\1\s\Xamarin.Forms.Xaml\XamlLoader.cs:89 
[mono]   at Xamarin.Forms.Xaml.XamlLoader.Load (System.Object view, System.Type callingType) [0x0002f] in D:\agent_work\1\s\Xamarin.Forms.Xaml\XamlLoader.cs:68 
[mono]   at Xamarin.Forms.Xaml.Extensions.LoadFromXaml[TXaml] (TXaml view, System.Type callingType) [0x00000] in D:\agent_work\1\s\Xamarin.Forms.Xaml\ViewExtensions.cs:36 
[mono]   at Contoso.Forms.Puppet.AppCenterContentPage.InitializeComponent () [0x00001] in /Users/guperrot/git/appcenter-sdk-dotnet/Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/obj/Debug/Contoso.Forms.Puppet.ModulePages.AppCenterContentPage.xaml.g.cs:38 
[mono]   at Contoso.Forms.Puppet.AppCenterContentPage..ctor () [0x00008] in /Users/guperrot/git/appcenter-sdk-dotnet/Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml.cs:19 
[mono]   at (wrapper managed-to-native) System.Reflection.MonoCMethod.InternalInvoke(System.Reflection.MonoCMethod,object,object[],System.Exception&)
[mono]   at System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters, System.Boolean wrapExceptions) [0x00005] in <d4a23bbd2f544c30a48c44dd622ce09f>:0 
[mono]    --- End of inner exception stack trace ---
[mono]   at (wrapper dynamic-method) System.Object.14(intptr,intptr,intptr)
[mono]   at (wrapper native-to-managed) System.Object.14(intptr,intptr,intptr)
```

Looks like horizontaltextalignment is an invalid value.
Please note that this app cannot be updated to Xamarin.Forms 3 (need to stay 2.5.x).